src/controller.cpp

```diff
@@ -1317,7 +1317,14 @@ void controller::edit_urls_file() {
 	v->push_empty_formaction();
 	stfl::reset();
 
-	utils::run_interactively(cmdline, "controller::edit_urls_file");
+	/* We should not check this particular exit code.
+	 * Editors usually fail when closing+saving (and stay open).
+	 * (tested with vim, nano, gedit, geany)
+	 * (texmaker 5.0.1 and notepadqq 1.0.1 close without neither showing an
+	 * error nor saving but despite return 0)
```

Comment:
I couldn't quite understand how different editors behave. Let's discuss this and figure out a better way to write this comment!

So, you're saying that editors are returning non-zero exit code when you close a document with unsaved changes, right? In other words, if the save dialog pops up when you try to close, the exit code will be non-zero?

And texmaker and notepadqq are different because they simply close, without popping up a save dialog, and they still return 0?